### PR TITLE
Update org.yaml to add SUSE team members contributing to Kubeflow

### PR DESCRIPTION
**Pull request by alfsuse**
We are starting to collaborate (intial effort around pipelines) at SUSE. This PR is to add the team members working there.

### CODE REVIEW

**Comment by Bobgy** [comment]
/assign @jlewi 
Can you please review this one? I'm not familiar this type of request

**Comment by jlewi** [comment]
@alfsuse Have you read the contributor guide? Could you please point at some PRs indicating your contributions?

**Comment by alfsuse** [comment]
> @alfsuse Have you read the contributor guide? Could you please point at some PRs indicating your contributions?

Sure thing @jlewi here it is for me: https://github.com/kubeflow/website/pull/2056 and for @gsantomaggio https://github.com/kubeflow/pipelines/pull/4278 as said in my initial PR we are starting to collaborate here and so more PR's will come from us.

File: github-orgs/kubeflow/org.yaml [hunk 1]
```diff
@@ -628,6 +628,12 @@ orgs:
             - tmckayus
             - pdmack
             privacy: closed
+          SUSE:
+            description: Team at SUSE contributing to Kubeflow.
+            members:
+            - alfsuse
```
Comment (jlewi) [inline github-orgs/kubeflow/org.yaml:634]: You'll need to add all members of the team to the organization as members.

Comment (alfsuse) [reply github-orgs/kubeflow/org.yaml:634]: @jlewi Added all members to the list in the last commit, we may eventually need to update the list in the future but I'll re-open another PR if that is the case.

**Comment by jlewi** [comment]
@alfsuse your diff doesn't look correct. I still don't see the folks added as members in the GitHub org. did you forget to add your changes?

**Comment by alfsuse** [comment]
@jlewi yep, I noticed that even if some PR's have been approved we still do not show up but I also see that the same PR's are not yet merged maybe that is why we are not yet there.  I would say let's wait until will be not completely merged, I'll re-check we show up in the contributor's list correctly and will comment on this PR.

**Comment by jlewi** [comment]
@alfsuse Which PR added your folks as members?

**Comment by alfsuse** [comment]
@jlewi  for @gsantomaggio : https://github.com/kubeflow/pipelines/pull/4278#  has been merged yesterday
For me, I'm currently waiting to merge: kubeflow/website#2056 .
Both of us have other PR's ready but still open. As I said we may wait to have more PR's merged and re-issue this one if you think is better.

**Comment by jlewi** [comment]
@alfsuse You have to update org.yaml to ensure that anyone who is listed in the team is also listed in the members section of the YAML. There is no connection to the other PRs.

**Comment by alfsuse** [comment]
Ah my bad and sorry okay should be fixed now... sorry about that.

**Comment by jlewi** [comment]
np
/lgtm
/approve

**Pull request merged**